data.json

```
@@ -694,5 +694,11 @@
     "githubUsername": "ChinmayMhatre",
     "imageUrl": "https://user-images.githubusercontent.com/51131670/111893373-5091f600-8a28-11eb-8e94-48b23eec0495.PNG",
     "issueNumber": 403
-  }
+  },
+   {
+    "name": "Sefat Siddiquea Sifa",
+    "githubUsername": "sifa123",
+    "imageUrl": "https://user-images.githubusercontent.com/63462931/112490528-a3add500-8da9-11eb-853e-9972b42e5caf.png",
+    "issueNumber": 410
+  },
```

There should be no comma here.  As Per [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas#trailing_commas_in_json). 
```suggestion
  }
```